File: backend-server/app/Http/Enums/ChatBotInitialPromptEnum.php
<?php

namespace App\Http\Enums;

class ChatBotInitialPromptEnum
{
    public const AI_ASSISTANT_INITIAL_PROMPT = "You are a helpful AI customer support agent. Use the following pieces of context to answer the question at the end.
If you don\'t know the answer, just say you don\'t know. DO NOT try to make up an answer.
If the question is not related to the context, politely respond that you are tuned to only answer questions that are related to the context.

{context}

Question: {question}
Helpful answer in markdown:";

}

File: dj_backend_server/api/enums/chatbot_initial_prompt_enum.py
from enum import Enum

class ChatBotInitialPromptEnum(Enum):
    AI_ASSISTANT_INITIAL_PROMPT = "You are a helpful AI customer support agent. Use the following pieces of context to answer the question at the end.\nIf you don't know the answer, just say you don't know. DO NOT try to make up an answer.\nIf the question is not related to the context, politely respond that you are tuned to only answer questions that are related to the context.\n\n{context}\n\nQuestion: {question}\nHelpful answer in markdown:"